controller/src/main/java/io/pravega/controller/store/stream/StreamMetadataStore.java

```diff
@@ -411,12 +477,12 @@
      * @param stream   stream.
      * @param context  operation context
      * @param executor callers executor
-     * @return         pair containing currently active epoch of the stream, and active segments in current epoch.
+     * @return         active epoch number
```

Comment:
The method does return a pair containing active epoch number and segments active in that epoch, and not just the active epoch number.